### ISSUE TITLE
Queue.poll() throws NPE

### ISSUE THREAD

**Issue by jankotek** [jankotek/mapdb#449]
```
	... 6 more
Caused by: java.lang.NullPointerException
	at org.mapdb.Queues$SimpleQueue.poll(Queues.java:135)
	at org.mapdb.Queues$SimpleQueue.take(Queues.java:240)
	at conc.Queue$2.run(Queue.java:41)
	at org.mapdb.Exec$1.run(Exec.java:28)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

```

    public static void main(String[] args) {

        DB db = DBMaker.newHeapDB().transactionDisable().make();

        final BlockingQueue q = db.createQueue("name", Serializer.STRING,true);

        new Thread(){
            @Override
            public void run() {
                for(long i=0;i<max;i++) {
                    q.add(i);
                }
            }
        }.start();

        Exec.execNTimes(10, new Runnable() {
            @Override
            public void run() {
                while(c.get()!=max){
                    try {
                        q.take();
                        c.incrementAndGet();
                    } catch (InterruptedException e) {
                        throw new RuntimeException(e);
                    }
                }
            }
        });

    }```

**Comment by jankotek** [comment]
feeder thread probably dies on class cast exception

**Comment by bwzhang2011** [comment]
@jankotek, any update with the issue.

**Comment by jankotek** [comment]
Could not replicate in 2.0, I believe it is already fixed. Not going to fix in 1.0
